youtube/youtube-extension-popup/src/Popup.js

```diff
@@ -0,0 +1,71 @@
+/* eslint-disable react/prop-types */
+import React from "react";
+import Modal from "./components/modal";
+export class Popup extends React.Component {
```

Comment:
- Leave a line before starting the component (after import statements)

- Change Class component to Functional component